apps/examples/security_test/Kconfig

```diff
@@ -0,0 +1,22 @@
+#
+# For a description of the syntax of this configuration file,
+# see misc/tools/kconfig-language.txt.
```

Comment:
We don't have `misc/tools/kconfig-language.txt`.

Reply:
Modified. Thanks.